src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,71 @@
-import React from 'react';
+import React, { ChangeEvent, FormEvent, useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  movies: Movie[];
+  setMovies: (movies: Movie[]) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ movies, setMovies }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [error, setError] = useState(false);
```

Comment:
if you store a `boolean` value - start name with `is`, `has`
```suggestion
  const [isError, setIsError] = useState(false);
```